db-service/lib/cqn2sql.js

```diff
@@ -664,7 +664,7 @@ class CQN2SQLRenderer {
     if ('param' in x) return wrap(this.param(x))
     if ('ref' in x) return wrap(this.ref(x))
     if ('val' in x) return wrap(this.val(x))
-    if ('xpr' in x) return wrap(this.xpr(x))
+    if ('xpr' in x) return wrap(`(${this.xpr(x)})`)
```

Comment:
Looks like far too much parenthesis

Reply:
I double checked and this is indeed not the correct place to do this. It is already being done where it should be done [here](https://github.com/cap-js/cds-dbs/blob/main/db-service/lib/cqn2sql.js#L684).

While constructing a reproduction query I found another issue.
```cql
-- CQL input
SELECT (1.0+2.0)*(1.0/2.0);
SELECT (1+2)*(1/2); -- is equal in cqn

-- SQL expectation
SELECT (1.0 + 2.0) * (1.0 / 2.0); -- 1.5

-- SQL output
SELECT (1 + 2) * (1 / 2); -- 0
```